tests/v2/validation/nodemetadata/nodemetadata.go

```diff
@@ -0,0 +1,61 @@
+package nodemetadata
+
+import (
+	"fmt"
+	"net/url"
+
+	v1 "github.com/rancher/shepherd/clients/rancher/v1"
+	coreV1 "k8s.io/api/core/v1"
+)
+
+func getNodeSpec(nodeList *v1.SteveCollection) (*coreV1.Node, error) {
+	firstNode := nodeList.Data[0]
```

Comment:
why not just use this as the argument sent to this function instead of the entire collection?

Reply:
Yes good point. But I want to understand how does it matter here?